### PR TITLE
RedGateSoftwareLtd.RedgateInstaller version 14.0

### PR DESCRIPTION
**Pull request by MEX-Kaz**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/91274)

### CODE REVIEW

**Comment by MEX-Kaz** [comment]
> @MEX-Kaz please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement

@microsoft-github-policy-service agree [company="{KazLabs}"]

**Comment by MEX-Kaz** [comment]
@microsoft-github-policy-service agree

File: manifests/r/RedGateSoftwareLtd/RedgateInstaller/14.0/RedGateSoftwareLtd.RedgateInstaller.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: RedGateSoftwareLtd.RedgateInstaller
+PackageVersion: 14.0
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://download.red-gate.com/installers/SQLDeveloperBundle/2022-12-08/SQLDeveloperBundle.exe
+  InstallerSha256: 9158E0F4B9F086202291C77F31154C341033B9B6DBE279706D9E20EF0CDE9D80
+  InstallerSwitches:
+    Silent: /IAgreeToTheEULA /q
```
Comment (Trenly) [inline manifests/r/RedGateSoftwareLtd/RedgateInstaller/14.0/RedGateSoftwareLtd.RedgateInstaller.installer.yaml:12]: This is a bit of a dangerous switch to add; Technically, this package should use the `Agreements` field if it requires EULA acceptance.

Comment (Trenly) [inline manifests/r/RedGateSoftwareLtd/RedgateInstaller/14.0/RedGateSoftwareLtd.RedgateInstaller.installer.yaml:12]: Please include `SilentWithProgress` switch

File: manifests/r/RedGateSoftwareLtd/RedgateInstaller/14.0/RedGateSoftwareLtd.RedgateInstaller.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: RedGateSoftwareLtd.RedgateInstaller
+PackageVersion: 14.0
+PackageLocale: en-US
+Publisher: Red Gate Software Ltd
+PackageName: SQL Developer Bundle
+License: Licensed
```
Comment (Trenly) [inline manifests/r/RedGateSoftwareLtd/RedgateInstaller/14.0/RedGateSoftwareLtd.RedgateInstaller.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

File: manifests/r/RedGateSoftwareLtd/RedgateInstaller/14.0/RedGateSoftwareLtd.RedgateInstaller.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: RedGateSoftwareLtd.RedgateInstaller
+PackageVersion: 14.0
+PackageLocale: en-US
+Publisher: Red Gate Software Ltd
+PackageName: SQL Developer Bundle
+License: Licensed
+Copyright: Copyright © Red Gate Software Ltd 2007-2021
+ShortDescription: SQLDevBundle
```
Comment (Trenly) [inline manifests/r/RedGateSoftwareLtd/RedgateInstaller/14.0/RedGateSoftwareLtd.RedgateInstaller.locale.en-US.yaml:11]: Please provide a better short description

File: manifests/r/RedGateSoftwareLtd/RedgateInstaller/14.0/RedGateSoftwareLtd.RedgateInstaller.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: RedGateSoftwareLtd.RedgateInstaller
+PackageVersion: 14.0
```
Comment (Trenly) [inline manifests/r/RedGateSoftwareLtd/RedgateInstaller/14.0/RedGateSoftwareLtd.RedgateInstaller.installer.yaml:5]: This doesn't correspond to any of the ARP Entries added by the installer. Either the primary entry needs to be added under `AppsAndFeaturesEntries` or this needs to be changed

File: manifests/r/RedGateSoftwareLtd/RedgateInstaller/14.0/RedGateSoftwareLtd.RedgateInstaller.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: RedGateSoftwareLtd.RedgateInstaller
+PackageVersion: 14.0
+PackageLocale: en-US
+Publisher: Red Gate Software Ltd
+PackageName: SQL Developer Bundle
```
Comment (Trenly) [inline manifests/r/RedGateSoftwareLtd/RedgateInstaller/14.0/RedGateSoftwareLtd.RedgateInstaller.locale.en-US.yaml:8]: This doesn't correspond to any of the ARP Entries added by the installer. Either the primary entry needs to be added under `AppsAndFeaturesEntries` or this needs to be changed

File: manifests/r/RedGateSoftwareLtd/RedgateInstaller/14.0/RedGateSoftwareLtd.RedgateInstaller.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: RedGateSoftwareLtd.RedgateInstaller
+PackageVersion: 14.0
+PackageLocale: en-US
+Publisher: Red Gate Software Ltd
```
Comment (Trenly) [inline manifests/r/RedGateSoftwareLtd/RedgateInstaller/14.0/RedGateSoftwareLtd.RedgateInstaller.locale.en-US.yaml:7]:
```suggestion
Publisher: Red Gate Software Ltd.
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Pull request closed without merging**